cinema/views.py

```diff
@@ -35,7 +41,7 @@ class GenreViewSet(
 ):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
-    authentication_classes = (TokenAuthentication,)
+
```

Comment:
The `permission_classes` attribute is correctly set to `IsAdminOrIfAuthenticatedReadOnly`, which aligns with the intended access control for this viewset.